### ISSUE TITLE
Skipping creating device XXX as no type yet for dim | onoff

### ISSUE THREAD

**Issue by autolog** [GledholtHall/beta-3#23]
Haviing installed pre-27, I deleted all my discovered devices and reran the discovery test,

This is the HE log for when discovery started:

> app:2982022-02-04 12:22:18.824 infoMQTT: |    ================== Startup complete ==================
> app:2982022-02-04 12:22:18.820 infoMQTT: |    Imported total [5] of [5] discovered
> app:2982022-02-04 12:22:18.817 infoMQTT: |    Imported [1] homie sensor devices of [1] discovered
> app:2982022-02-04 12:22:18.813 infoMQTT: |    Imported [3] homie dim devices of [3] discovered
> app:2982022-02-04 12:22:18.809 infoMQTT: |    Imported [1] homie onoff devices of [1] discovered
> app:2982022-02-04 12:22:18.805 infoMQTT: |    [23]  Hubitat endpoints enabled on MQTT
> app:2982022-02-04 12:22:18.801 infoMQTT: |    [22]  Hubitat devices are enabled on MQTT
> app:2982022-02-04 12:22:18.796 infoMQTT:          [22] Hubitat 'everything' all capability devices enabled on MQTT
> app:2982022-02-04 12:22:18.779 infoMQTT: |    ==================================================
> app:2982022-02-04 12:22:18.769 infoMQTT: Initialise exit
> app:2982022-02-04 12:22:18.746 infoMQTT: Subscribing to 0 individual device topics
> app:2982022-02-04 12:22:18.743 infoMQTT: Subscribing to virtuals
> app:2982022-02-04 12:22:18.725 infoMQTT: No Discovery startups enabled
> app:2982022-02-04 12:22:18.721 infoMQTT: HE device publishing to MQTT has completed in 15 secs
> .
> .
> .
> .
> 
> app:2982022-02-04 12:22:02.669 warnMQTT: Unknown property value pressure
> app:2982022-02-04 12:22:02.296 warnMQTT: Unknown property value pressure
> app:2982022-02-04 12:22:02.141 warnMQTT: Unknown property value checkInterval
> app:2982022-02-04 12:22:01.831 warnMQTT: Unknown property value ultravioletIndex
> app:2982022-02-04 12:22:01.799 warnMQTT: Unknown property value tamper
> app:2982022-02-04 12:22:01.767 warnMQTT: Unknown property value powerSource
> app:2982022-02-04 12:22:01.310 warnMQTT: Unknown property value pressure
> dev:4362022-02-04 12:22:00.899 warnMQTT> ## Restarted ##
> dev:4362022-02-04 12:22:00.895 infoMQTT> Log level set to 2
> app:2982022-02-04 12:21:50.597 infoMQTT: MQTT reports status of Connect
> dev:4362022-02-04 12:21:50.573 infoMQTT> Connected to MQTT broker tcp://192.168.1.8:1883
> dev:4362022-02-04 12:21:49.566 infoMQTT> Broker: Status: Connection succeeded
> dev:4362022-02-04 12:21:49.244 infoMQTT> Connecting with 'clean' session as Hubitat_home-1 to MQTT broker tcp://192.168.1.8:1883
> dev:4362022-02-04 12:21:49.239 infoMQTT> MQTT> client beta 3e pre 26 home-1
> dev:4362022-02-04 12:21:49.206 infoMQTT> HAIn Discovery is false
> dev:4362022-02-04 12:21:49.195 infoMQTT> Resetting MQTT connection
> dev:4362022-02-04 12:21:49.191 infoMQTT> Disconnected MQTT
> app:2982022-02-04 12:21:49.077 infoMQTT: Waiting for MQTT connection: false
> app:2982022-02-04 12:21:49.010 infoMQTT: Standard Logging
> app:2982022-02-04 12:21:48.940 infoMQTT: initialising with App count 0
> app:2982022-02-04 12:21:48.906 infoMQTT: Hubitat hub name is : home-1
> app:2982022-02-04 12:21:48.897 infoMQTT: MQTT Client driver present
> app:2982022-02-04 12:21:48.795 infoMQTT: Removing unused atomicState variables
> app:2982022-02-04 12:21:48.427 infoMQTT: MQTT Updated
> app:2982022-02-04 12:21:38.598 warnMQTT: Skipping creating device dev-99417865 as no type yet for dim
> app:2982022-02-04 12:21:38.546 warnMQTT: Skipping creating device dev-897702736 as no type yet for dim
> app:2982022-02-04 12:21:38.474 warnMQTT: Skipping creating device dev-1403695954 as no type yet for dim
> app:2982022-02-04 12:21:38.395 warnMQTT: Skipping creating device dev-1333137217 as no type yet for onoff
> app:2982022-02-04 12:21:38.390 warnMQTT: Already in dropdown list:OnOff dev-1333137217
> dev:4362022-02-04 12:21:38.384 infoMQTT> MQTT subscribing to: homie/indigo-1/dev-1921277469/$type
> dev:4362022-02-04 12:21:38.369 infoMQTT> MQTT subscribing to: homie/indigo-1/dev-99417865/$type
> dev:4362022-02-04 12:21:38.345 infoMQTT> MQTT subscribing to: homie/indigo-1/dev-897702736/$type
> dev:4362022-02-04 12:21:38.331 infoMQTT> MQTT subscribing to: homie/indigo-1/dev-1403695954/$type
> dev:4362022-02-04 12:21:38.316 infoMQTT> MQTT subscribing to: homie/indigo-1/dev-1333137217/$type
> dev:4362022-02-04 12:21:38.306 infoMQTT> homie discovery is for device: indigo-1

However, no devicse are created including the sensor.

Going back into the MQTT App, there appaers to be a discrepancy in the count of enabled devices:
![Screenshot 2022-02-04 at 12 28 12](https://user-images.githubusercontent.com/11138074/152529209-59e38d07-8843-4179-a142-0ab31ebd6edd.png)

So in summary, none of the devices discovered are now being created in pre-27.

**Comment by GledholtHall** [comment]
Ooops - that’s what happens at 7am I guess
I’ll take a look - .. after my porridge ;-)

**Comment by GledholtHall** [comment]
Did you delete the indigo-1 device on my broker that these examples came from ? I was hoping to test against that ..

**Comment by autolog** [comment]
No, I didn't

**Comment by autolog** [comment]
I can add it back in again

**Comment by GledholtHall** [comment]
Fixed in pre27a I hope
